.github/workflows/ci.yml

```diff
@@ -20,9 +20,9 @@ jobs:
       - name: Install snap
         run: sudo apt install snapd
       - name: Install heroku cli
-        run: sudo snap install --classic heroku
-        env:
-          HOME: ${GITHUB_WORKSPACE}
+        uses: wei/curl@master
```

Comment:
I would avoid using Actions that almost does nothing (look here what it does: https://github.com/wei/curl/blob/master/entrypoint.sh). Why not using curl directly since it already is inside ubuntu? We had this problem with heroku because an Action was used for heroku when it was not necessary, so better avoid it when you can.

Reply:
I have revert this change, and created another branch with the improvements of the git actions